src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -74,7 +74,7 @@ export type Props = {
   experimentationId: string;
   elapsedTimeInDaysSinceInitialScan?: number;
   totalNumberOfPerformedScans?: number;
-  activeTab: TabType;
+  activeTab: string;
```

Comment:
No longer marking this as `TabType` was surprising - is that because we the path parameter in `page.tsx` could be any string? The safest solution IMHO would be to have an `isValidTabType` [user-defined type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) (essentially, extracting [this if condition](https://github.com/mozilla/blurts-server/blob/c8a4edf1a5bbe238948cb6ca40af791fe2fd3b15/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/%5B%5B...slug%5D%5D/page.tsx#L61-L64) into a function), or otherwise at least to add a type assertion in there, rather than having the type information say that this component is fine with any string.

Reply:
I’ve added a type assertion in https://github.com/mozilla/blurts-server/pull/4477/commits/3b355f3ea16e5311659fda27d6daaf494ce34292.